finder-app/writer.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+echo "Checking for arguments..."
+if [ $# -lt 2 ]; then
+    echo "Error : Arguments not specified correctly!!!"
+    exit 1
+else
+    echo "Pass"
+fi
+
+writefile=$1
+writestr=$2
+
+# Set / as delimiter to split string
+IFS='/'
+read -a strarr <<< "$writefile"
+
+# To create directory if it doesn't exists
+for (( n=0; n < ${#strarr[*]}-1; n++))
+do
+  writedir+="${strarr[n]}/"
+done
```

Comment:
@ameyflash I don't think you need lines 15 through 22, `mkdir -p` should handle this for you I believe.